config.template/faf-clans/faf-clans.env

```diff
@@ -1,7 +1,3 @@
-API_BASE_URL=https://api.dev.faforever.com
-OAUTH_CLIENT_ID=83891c0c-feab-42e1-9ca7-515f94f808ef
-OAUTH_REDIRECT_URI=https://clans.faforever.com
-
-#VIRTUAL_HOST=clans.faforever.com
-#LETSENCRYPT_HOST=clans.faforever.com
-#LETSENCRYPT_EMAIL=admin@faforever.com
+API_BASE_URL=https://localhost:8010
+OAUTH_CLIENT_ID=11111111-2222-3333-4444-555555555555
+OAUTH_REDIRECT_URI=https://localhost
```

Comment:
Does that work?
This makes you being redirected to `http://localhost:`. And there you get a connection refused. Is that what you want?
So it should probably be `http://localhost:8096`